server/keyspace/keyspace.go

```diff
@@ -98,14 +106,22 @@ func (manager *Manager) Bootstrap() error {
 		return err
 	}
 	now := time.Now().Unix()
+	id, err := manager.kgm.GetAvailableKeyspaceGroupIDByKind(endpoint.Basic)
+	if err != nil {
+		return err
+	}
 	defaultKeyspace := &keyspacepb.KeyspaceMeta{
 		Id:             DefaultKeyspaceID,
 		Name:           DefaultKeyspaceName,
 		State:          keyspacepb.KeyspaceState_ENABLED,
 		CreatedAt:      now,
 		StateChangedAt: now,
+		Config: map[string]string{
```

Comment:
How do we add UserKind to the existing keyspaces? I didn't see they have UpdateKeyspace API. UpdateKeyspaceState API doesn't apply.

Reply:
Regarding group id, my understanding is that AddKeyspacesToGroups() API will do the assignment and ad d group id to the existing keyspaces, which is ok.

Reply:
Will AddKeyspacesToGroups be exposed?